.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: python
 python:
-    - "2.7"
-    - "3.6"
     - "3.7"
+    - "3.10"
```

Comment:
any reason not to add 3.8 and 3.9 while we're at it?

Reply:
As I say below, I want to minimise the real changes. I've now dropped Python 3.10 from the test matrix in this PR